plugins/dominant-color-images/hooks.php

```diff
@@ -186,3 +186,76 @@ function dominant_color_render_generator(): void {
 	echo '<meta name="generator" content="dominant-color-images ' . esc_attr( DOMINANT_COLOR_IMAGES_VERSION ) . '">' . "\n";
 }
 add_action( 'wp_head', 'dominant_color_render_generator' );
+
+/**
+ * Adds inline CSS for dominant color styling in the WordPress admin area.
+ *
+ * This function registers and enqueues a custom style handle, then adds inline CSS
+ * to apply background color based on the dominant color for attachment previews
+ * in the WordPress admin interface.
+ *
+ * @since 1.1.3
```

Comment:
Just in case, these could be set to n.e.x.t. and auto-updated to the correct version when this is packaged in a release:

```suggestion
 * @since n.e.x.t
```

🔢 Same throughout